api/KS.FiksProtokollValidator.WebAPI/Controllers/TestCasePayloadFilesController.cs

```diff
@@ -198,13 +201,15 @@ public async Task<ActionResult> UploadCustomPayload(string testcaseId)
             return new OkResult();
         }
         
-        // GET api/<TestsCasePayloadFilesController>/TestCaseName/Attachement/attachmentFileName
-        [HttpGet("{protocol}/{testCaseName}/Attachement/{attachmentFileName}")]
-        public ActionResult GetAttachmentPayloadFile(string protocol, string testCaseName, string attachmentFileName)
+        // GET api/<TestsCasePayloadFilesController>/testCaseId/Attachement/attachmentFileName
+        [HttpGet("{testCaseId}/Attachment/{attachmentFileName}")]
```

Comment:
Takk for at du fikset den trykkleifen "attachement" også. Rykket litt i OCD'en min når jeg så den :D